demos/src/aside.mustache

```diff
@@ -0,0 +1,41 @@
+<article class="o-grid-row demo demo-article">
+	<div data-o-grid-colspan="12 S11 Scenter M9 L8 XL7">
+		<aside class="demo-article__body o-typography-aside-wrapper">
+			<h3>This is the aside title</h3>
+			<article>
+				<h4>
+					<a href="#">Headline link to another article</a>
+				</h4>
+			</article>
+			<div>
+				<p>Here's some content in the aside.</p>
+				<p>There could be a few paragraphs.</p>
+			</div>
+			<blockquote>
+				<p>Here's a blockquote that's pulled out into an aside</p>
+				<footer>This is who said it</footer>
+			</blockquote>
+		</aside>
+		<br>
+		<aside>
+			<div class="o-typography-big-number">222m</div>
```

Comment:
What @andygnewman and I discussed was him updating o-typography with this, and asides and then we could see where there was opportunity to move stuff out. So the next stage would be to update o-big-number, and create o-aside from here. This is the interim.